doc/repofind.md

```diff
@@ -0,0 +1,21 @@
+# repofind
+
+## Name
+
+repofind - Display information on file:// repositories
+
+## Synopsis
+
+repofind [-isu ARG] [--] ARGS...
+
+## Description
+
+## Operations
+
+* __-i__ Find and list all file:// repos in pacman.conf
+* __-s__ Interactively select a repo to use
```

Comment:
repofind -s gives you a selection dialog, and prints the result as "name
location" on stdout. So, "to use" doesn't make sense in this context (though it does when using repofind -s in other scripts, such as aursync).